src/strawchemy/utils/strawberry.py

```
@@ -44,15 +44,17 @@ def dto_model_from_type(type_: Any) -> Any:
     return type_.__dto_model__
 
 
-def strawberry_contained_types(type_: StrawberryType | Any) -> tuple[Any, ...]:
+def strawberry_contained_types(type_: StrawberryType | Any, resolve_lazy: bool = True) -> tuple[Any, ...]:
```

Ruff (0.15.15)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `type_`

(ANN401)